main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -51,6 +51,9 @@ class Flix {
     */
   private val internals = List(
     "Prelude.flix" -> StreamOps.readAll(LocalResource.Library.Prelude),
+    "Option.flix" -> StreamOps.readAll(LocalResource.Library.Option),
```

Comment:
It is kind of complicated. 

Ideally and ultimately we want the library to be included by default. So this would be the right place to put them. On the other hand, we do have 2000+ tests which would mean that the library would be included by default in all those tests which would slow down unit testing significantly.

I  think we need to perform an experiment to discover how much slow down is there, and whether we can include the library by default for the tests.